cookbooks/web/files/default/static/funding.json

```diff
@@ -76,22 +75,45 @@
     ],
     "plans": [
       {
-        "guid": "one-time-donation",
+        "guid": "one-time-donation-usd",
         "status": "active",
-        "name": "One-time donation",
+        "name": "One-time USD donation",
         "amount": 0,
         "currency": "USD",
         "frequency": "one-time",
         "channels": [
           "funding-website",
-          "bitcoin",
-          "eur-account",
           "usd-account",
-          "gbp-account",
           "paypal",
           "gh-sponsors"
         ]
       },
+      {
+        "guid": "one-time-donation-gbp",
+        "status": "active",
+        "name": "One-time GBP donation",
+        "amount": 0,
+        "currency": "USD",
+        "frequency": "one-time",
+        "channels": [
+          "funding-website",
+          "gbp-account",
+          "paypal"
+        ]
+      },
+      {
+        "guid": "one-time-donation-eur",
+        "status": "active",
+        "name": "One-time EUR donation",
+        "amount": 0,
+        "currency": "USD",
```

Comment:
Should this be “EUR”?

Reply:
I just fixed it: https://github.com/openstreetmap/chef/commit/672e2c86381b6393c7c43f836d24ccc5372ddb74